package.json

```diff
@@ -1,11 +1,17 @@
 {
   "name": "jStorage",
```

Comment:
So to begin with I just started with the core package definition. Uploading packages without talking to you seemed like a stupid thing, that's why I wanted to talk the past days ;)